src/Conversion/ONNXToMhlo/CMakeLists.txt

```diff
@@ -52,3 +64,7 @@ add_onnx_mlir_library(OMONNXToMhlo
 target_link_libraries(MhloDialect PUBLIC
   MLIRSparseTensorDialect
   )
+
+target_link_libraries(StablehloTypeInference PUBLIC
```

Comment:
Why is this needed? The two libs are both linked into the target above

Reply:
But `StablehloTypeInference` links into `StablehloBase`, which is missing in `mlir-hlo` repo.

Reply:
Have you let the mhlo folks know, so that it can be fixed there as well?

Reply:
It is added. https://github.com/openxla/stablehlo/pull/456.

BTW, I find that the additional linking relationship below is already added in mlir-hlo repo.
```
target_link_libraries(MhloDialect PUBLIC
  MLIRSparseTensorDialect
  )
```